src/app/static/src/app/components/plots/choropleth/Choropleth.vue

```diff
@@ -59,7 +60,7 @@
 
     interface Data {
         style: any,
-        fullIndicatorRanges: Dict<NumericRange>
+        fullIndicatorRanges: Dict<NumericRange>,
```

Comment:
ive seen this type of change in a couple places in your PRs, i dont mind it at all but just curious, is this an auto formatter?

Reply:
ohhh okay makes sense haha, yh no worries im not fussed about whether it is there or not